spec/requests/stories_index_spec.rb

```diff
@@ -16,11 +16,11 @@
       article = create(:article, featured: true)
 
       get "/"
-
       expect(response.body).to include(CGI.escapeHTML(article.title))
     end
 
     it "renders proper description" do
+      allow(SiteConfig).to receive(:community_description).and_return("Some description")
```

Comment:
I wonder if we should move some of the more common ones into a global spec config, because I'm afraid that otherwise too many people will have errors that are hard to figure out when adding new tests. WDYT @Ridhwana?

Reply:
`community_description` is stubbed in more than one place and maybe it would make sense to stub `logo_png` too. But I guess we can do this at a later point too.

Reply:
💯 I moved community_description into the rails helper, but I kept the suggested tags and the on boarding stuff relevant to their files. I'm going to keep an eye on logo_png before I add it to the global config